kubernetes/client/api_client.py

```diff
@@ -494,6 +497,8 @@ def select_header_content_type(self, content_types):
 
         if 'application/json' in content_types or '*/*' in content_types:
             return 'application/json'
+        elif self.prefered_content_type in content_types:
+            return self.prefered_content_type
```

Comment:
This solves our problem. I'm not sure if this is general for other project as well. It works for us because PATCH is so far the only verb that doesn't take `application/json` and `*/*` as content type.

Reply:
I've removed prefered_content_type.